x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -5,6 +5,7 @@
  * 2.0.
  */
 
+export const FINDINGS_STATUS_ROUTE_PATH = '/internal/cloud_security_posture/findings_status';
```

Comment:
I would call it `status` or `info`. It is not necessarly a status on the findings as it is a general status